src/domain/limit/getStorageStatus.js

```diff
@@ -34,11 +35,11 @@ async function formatClaims(invokerAddress, contractAddress, claimsMap, removeCa
 }
 
 async function getStorageStatus({ contractAddress, invokerAddress, setCache = false }) {
-  const limit = await contractAddress ? Limit.findOne({ contractAddress }) : Limit.findOne({ invokerAddress });
+  const limit = await Limit.findOne({ invokerAddress, contractAddress });
```

Comment:
This is wrong . same condition you need to take care of older portals where invoker address is not set